Find a functionally equivalent code:android.content.pm.PermissionInfo.protectionToString:COMMENT
Method Modifier: public      static      hidden      
Comment:/**
 * @hide
 */

@@ -20,24 +20,27 @@
     if ((level & PermissionInfo.PROTECTION_FLAG_DEVELOPMENT) != 0) {
         protLevel += "|development";
     }
+    if ((level & PermissionInfo.PROTECTION_FLAG_APPOP) != 0) {
+        protLevel += "|appop";
+    }
     return protLevel;
 }

